Lab 4 AT 9-17-21.Rmd

```diff
@@ -140,12 +141,13 @@ leaflet(met_avg) %>%
 Greater relative humidity is observed on the east coast than the west coast.
 
 8. Use a ggplot extension
-```{r}
+```{r eval = FALSE}
```

Comment:
I set `eval = FALSE` b/c I was not able to install this ggplot2 extension! You can set it to `TRUE` whenever you want, @amyt917.